subtasks/text_task/lm_poem.yaml

```diff
@@ -0,0 +1,14 @@
+# Adapter-Hub subtask definition
```

Comment:
Can we add a URL or/and a citation here to clearify the exact poem dataset used? (cf. https://github.com/Adapter-Hub/Hub/blob/master/TEMPLATES/subtask.template.yaml)